circle.yml

```diff
@@ -18,7 +18,7 @@ jobs:
           name: Build docker image and push to repo
           command: |
             docker version
-            docker build -t app:build -f Dockerfile.deploy .
+            docker build -t app:build -f Dockerfile.py3.deploy .
```

Comment:
These are used for addons-dev.allizom.org

Reply:
yep.  The commit is misnamed - the issue was about having addons-dev run python3 also.